blind_walking/envs/env_builder.py

```diff
@@ -55,7 +55,14 @@ def build_regular_env(robot_class,
 
   env_sensor_list = [
     environment_sensors.LastActionSensor(num_actions=a1.NUM_MOTORS),
+<<<<<<< HEAD
```

Comment:
I think you didn't resolve the mere conflicts

Reply:
thanks for catching that! my bad.